cpp/scenario/gazebo/include/scenario/gazebo/Link.h

```diff
@@ -68,8 +68,15 @@ class scenario::gazebo::Link
     // Link Core
     // =========
 
-    std::string name() const;
+    /**
+     * Get the name of the link.
+     *
+     * @param scoped Return the scoped link name.
```

Comment:
The docs say it return the scoped name, but the default value for scoped is `false` .

Reply:
Good catch! Thanks

Reply:
Addressed in https://github.com/robotology/gym-ignition/compare/c3162c59ecfd0fa84573201c16f33050ed4eb383..2fbe4410c7caf590d78e5008106321c829d7acd8.